src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.product-card {
+  box-sizing: border-box;
+  padding-inline: 16px;
+  width: 200px;
+  background-color: $main-background-color;
```

Comment:
Ensure that the variable `$main-background-color` is defined in your SCSS files. Otherwise, this will cause a compilation error.